### PR TITLE
[14.0][MIG] sale_report_margin: Migration to 14.0

### PR DESCRIPTION
**Pull request by ferran-S73**
Standard migration

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

**Comment by ferran-S73** [comment]
@sergio-teruel @chienandalu could you please review this PR?

File: sale_report_margin/__manifest__.py [hunk 1]
```diff
@@ -2,8 +2,9 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Sale Report Margin",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "summary": "Adds purchase_price field to sale report",
```
Comment (rousseldenis) [inline sale_report_margin/__manifest__.py]: This is more a technical summary than a functional one (should be understandable by everyone).

Comment (ferran-S73) [reply sale_report_margin/__manifest__.py]: Updated and rebased too. Hope it's more understandable now.

**Comment by ferran-S73** [comment]
Hey @rousseldenis @CarlosRoca13 @sergio-teruel @chienandalu this has been merged for 15.0, any chance to merge this PR? Thanks

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**